checker/jdk/nullness/src/java/lang/reflect/Constructor.java

```diff
@@ -6,7 +6,7 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
-public final class Constructor<T> extends AccessibleObject implements GenericDeclaration, Member {
+public final class Constructor<T extends @Nullable Object> extends AccessibleObject implements GenericDeclaration, Member {
```

Comment:
We don't have a test case that demonstrates the need for this, but I agree it's a good idea for consistency.  I will make the change and push it after local tests pass.